src/core/url-beautifier/details/parser.ts

```diff
@@ -28,7 +28,7 @@ export class DetailsUrlParser extends UrlParser<UrlBeautifier.DetailsUrlState> {
       }
     }
 
-    return { variants, data: { id, title } };
+    return { variants, data: { id, title }, id };
```

Comment:
rather than pass the `id` in two places here, can we have the logic than handles this return value expect to pull the `id` value from the `data` field?

Reply:
This was something we wanted to discuss. Currently we just have 2 ids to prevent similar issues from happening in other locations.

Reply:
I looked through the code, this makes more sense to me now 👍